.github/workflows/build.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: Build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.0.0
+          bundler-cache: true
+      - name: standardrb
+        run: bundle exec standardrb
```

Comment:
I would add a newline at the end of this file and then set up your editor to automatically do that to all files. That would prevent this annoying red warning circle.